framework/models.py

```diff
@@ -0,0 +1,69 @@
+# External Libraries
+from pony.orm import Set, Database, Required, PrimaryKey, db_session, Optional
+
+db = Database()
+
+
+class Base:
+    @classmethod
+    @db_session
+    def exists(cls, id_: str):
+        return cls.get(id=id_) is not None  # pylint: disable=no-member
+
+    @classmethod
+    @db_session
+    def get_s(cls, arg):
+        return cls.get(id=arg)  # pylint: disable=no-member
+
+    @property
+    @db_session
+    def json(self):
+        return self.__class__[self.id].to_dict(with_collections=True, exclude="password")  # pylint: disable=no-member
+
+
+class Mod(db.Entity, Base):
+    id = PrimaryKey(str)
```

Comment:
IDs should probably be an int as Python and (most) DBs can support long numbers, and it can be coerced to strings when needed sending data from the API.

Reply:
Keeping it as string to make other languages (e.g. JS) able to use the database if needed as large integers can be inconsistent.